main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
I believe these are comments in Javascript, but we're writing in Python so it won't be read as a comment. The syntax for comments in Python is `#`. This also occurs in lines 5, 12, 22.